packages/osintgram/PKGBUILD

```diff
@@ -0,0 +1,49 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=osintgram
+pkgver=348.50d44fd
+pkgrel=1
+pkgdesc='OSINT tool on Instagram. It offers an interactive shell to perform analysis on Instagram account of any users by its nickname.'
+arch=('any')
+groups=('blackarch' 'blackarch-recon')
+url='https://github.com/Datalux/Osintgram'
+license=('GPL3')
+depends=()
+makedepends=('git' 'python')
+source=("git+https://github.com/Datalux/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+prepare() {
+  cd $pkgname
+
+  python -m venv venv
```

Comment:
don't use a virtual environment, we use those only for very complex cases

all the dependencies in `requirements.txt` are available in AL or BA repo outside of `instagram-private-api` that need to be packaged as well in your PR.

Reply:
`prepare()` deleted.